lib/src/otagrum/JunctionTreeBernsteinCopula.hxx

```diff
@@ -0,0 +1,136 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The JunctionTreeBernsteinCopula distribution
+ *
+ *  Copyright 2005-2017 Airbus-EDF-IMACS-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
```

Comment:
Could you use the same headers as in master ? The date, authors and license is a bit different (2018/ LIP6/ GPL)

Reply:
done